modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectValidationRuleLocalServiceTest.java

```diff
@@ -177,6 +186,101 @@ public void testAddObjectValidationRule() throws Exception {
 						RandomTestUtil.randomString()
 					).build())));
 
+		ObjectField textObjectField = _objectFieldLocalService.fetchObjectField(
+			_objectDefinition.getObjectDefinitionId(), "textObjectField");
+
+		List<ObjectValidationRuleSetting> objectValidationRuleSettings =
+			new ArrayList<>();
```

Comment:
@pedro-oliveira446 please move the `textObjectField` variable as close as possible to its first usage, i.e., before the `CompositeKeyMustHaveMinObjectFields`